### PR TITLE
Set default missile acceleration to 0

### PR DESCRIPTION
**Pull request by cjshmyr**
As discussed on IRC and proven out in #10129. If merged, we should make a ticket for `next+1` on the topic of re-adding acceleration, since we're short on time.

### CODE REVIEW

**Comment by GraionDilach** [comment]
This is stupid. Disabling `Acceleration` but keeping the `MaximumSpeed` > `MaximumLaunchSpeed` > `MinimumLaunchSpeed` defaults are just bogus and definitely far from elegant.

**Comment by GraionDilach** [comment]
I did a quick glampse in the missile yamls -__ this is really bad__. Most missiles does not override the `LaunchSpeeds` so they will stay varied, which means missiles might end up being launched with a low speed without catching up to their old speed.

Set the _MaximumSpeed_ default to _MaximumLaunchSpeed_ instead of disabling Acceleration for the effect.

**Comment by penev92** [comment]
>This is stupid.

No, this is changing a dice-roll-generated default value to a sensible one. It's not disabling `Acceleration`, it's setting a proper default to avoid side-effects like those that caused the issue to be opened in the first place. From there any weapon can set its own acceleration if it needs it.

But you are right, there are a lot more apparently random generated default values in that class. In any case, this is probably good enough to fix the issue for the release. We can do more fine tuning (and hopefully fixing those ridiculous defaults) and reintroducing acceleration to RA missiles, because it looks pretty good, for the next release.

File: OpenRA.Mods.Common/Effects/Missile.cs [hunk 1]
```diff
@@ -50,7 +50,7 @@ public class MissileInfo : IProjectileInfo
 		public readonly WDist MaximumSpeed = new WDist(384);
 
 		[Desc("Projectile acceleration when propulsion activated.")]
-		public readonly WDist Acceleration = new WDist(5);
+		public readonly WDist Acceleration = new WDist(0);
```
Comment (pchote) [inline OpenRA.Mods.Common/Effects/Missile.cs]: `WVec.Zero` please

**Comment by cjshmyr** [comment]
> I did a quick glampse in the missile yamls -__ this is really bad__. Most missiles does not override the LaunchSpeeds so they will stay varied, which means missiles might end up being launched with a low speed without catching up to their old speed.

Just the TS/RA2/heightmap mods would have an issue with perma-slow missiles due to no acceleration. The minimum speed is only ever used with regards to heightmaps.

Personally I would want all the defaults to be for an accelerated missile, we'd need less duplication in yaml, and that's I think the end goal. Point of this patch was just to be a band-aid for prep (TS is not shipped there), so I'll need to retarget the merge branch with a new PR. We'd be following up on bleed with a more proper fix in the future.

**Pull request closed without merging**